plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -88,6 +88,11 @@ var BasicsChart = React.createClass({
     patientData.basicsData = this.state;
     this.props.updateBasicsData(patientData);
   },
+  addToBasicsData: function(key, value) {
```

Comment:
Move this to the basicsActions please, for consistency. And then it doesn't need to be passed down through props but rather you can require basicsActions where it's needed. (We'll be changing this strategy once we're redux from top-to-bottom, but for now I'd rather keep internal consistency within the other basics actions, limited though they may be.)